serverless.ts

```diff
@@ -468,7 +503,26 @@ const serverlessConfig: AWS = {
       ],
       environment: {
         ALGOLIA_API_KEY: `\${ssm:algolia-index-api-key-${envAlias}}`,
-        ALGOLIA_INDEX: `asap-hub_research_outputs_${envRef}`,
+        ALGOLIA_INDEX: `asap-hub_${envRef}`,
```

Comment:
well thats not quite right. You are switching to the new index which means we will no longer sync the old one, i think this should be done once we move over @marot might want to take a look at this